prime-router/metadata/hl7_mapping/ORU_R01/base/observation-result.yml

```diff
@@ -154,6 +154,10 @@ elements:
     value: [ '%resource.coding.code' ]
     hl7Spec: [ '%{hl7OBXField}-15-3' ]
 
+  - name: observation-method
```

Comment:
I assume this is a required field, even though we are filling it with blank string?

Reply:
Not required, just an empty element that we can override in the CO schema